application-blog-ui/src/main/resources/Blog/BlogStyle.xml

```diff
@@ -226,6 +226,7 @@
   float: right;
   position: relative;
   z-index: 2;
+  margin-left: 1.5em;
```

Comment:
Looks good to me, thanks you :)
Maybe you should add margin-bottom too, because the second line of content can be long too. What do you think?